app/code/Magento/Sitemap/Model/Observer.php

```diff
@@ -67,13 +69,21 @@ class Observer
     protected $inlineTranslation;
 
     /**
+     * @var \Magento\Store\Model\App\Emulation $appEmulation
+     */
+    protected $appEmulation;
+
+    /**
+     * Observer constructor.
+     * @param Emulation|null $appEmulation
      * @param \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
-     * @param \Magento\Sitemap\Model\ResourceModel\Sitemap\CollectionFactory $collectionFactory
+     * @param ResourceModel\Sitemap\CollectionFactory $collectionFactory
      * @param \Magento\Store\Model\StoreManagerInterface $storeManager
      * @param \Magento\Framework\Mail\Template\TransportBuilder $transportBuilder
      * @param \Magento\Framework\Translate\Inline\StateInterface $inlineTranslation
      */
     public function __construct(
+        Emulation $appEmulation = null,
```

Comment:
Please add move the new parameter to the last position in the parameters list

Reply:
ok.